README.md

```diff
@@ -52,7 +52,7 @@ impl From<Data> for DataSerde {
     }
 }
 
-#[cxx_qt::bridge]
+#[cxx_qt::bridge(namespace = "cxx_qt::my_object")]
```

Comment:
Need to document the namespace in the book and state how currently each QObject needs it's own namespace otherwise there will be collisions until we get a bit further....

Reply:
Added some minimal documentation around this for now.